assets/chessground.base.css

```diff
@@ -173,52 +173,52 @@ piece.fading {
   transform: translateY(39%);
 }
 
-.cg-wrap coords.squares {
+coords.squares {
   bottom: 0;
   left: 0;
   text-transform: uppercase;
   text-align: right;
   flex-flow: column-reverse;
   height: 100%;
   width: 12.5%;
-}
 
-.cg-wrap coords.squares.black {
-  flex-flow: column;
-}
+  &.black {
```

Comment:
This is pure CSS, so we can't use these kinds of shortcuts.

Reply:
fixed